lib/webdriveragent.js

```diff
@@ -182,7 +182,9 @@ class WebDriverAgent {
         return;
       }
       if (isPertinentLogLine(out)) {
-        agentLog.debug(`${prefix}: ${out.trim()}`);
+        for (let line of out.split("\n")) {
+          agentLog.debug(`${prefix}: ${line.trim()}`);
```

Comment:
I will add that we should probably also check if there is anything in the line. We sometimes get empty lines, which do not need to be printed.